res/sample/configuration.toml

```diff
@@ -24,44 +24,44 @@
     # dynamic changes from Consul.
     [Writable.Pipeline.Functions.FilterByProfileName]
       [Writable.Pipeline.Functions.FilterByProfileName.Parameters]
-        # TODO: Remove 'Unknown' once V1 events no longer supported.
-        ProfileNames = "Random-Float-Device, Random-Integer-Device, Unknown"
-        FilterOut = "false"
+      # TODO: Remove 'Unknown' once V1 events no longer supported.
```

Comment:
so we still support V1?

Reply:
Yes, just unit Device Virtual has been updated to V2 and generates V2 DTOs.